src/cobra/summary/metabolite_summary.py

```diff
@@ -0,0 +1,399 @@
+"""Provide the metabolite summary class."""
+
+
+import logging
+from operator import attrgetter
+from textwrap import shorten
+from typing import TYPE_CHECKING, List, Optional, Union
+
+from pandas import DataFrame
```

Comment:
Any specific reason of using `from pandas import DataFrame` and not `import pandas as pd`? I find the latter to be more explicit about the fact that `DataFrame` comes from an external library and not something from our code base.

Reply:
Personal style. I can change it to what you suggest.

Reply:
I really prefer the latter one. Also, it's also the one which the scientific community in general likes, since most of the code in the wild that I have encountered, uses this. Can we please make the change?

Reply:
Done.